test/perf_script.ml

```diff
@@ -41,6 +41,8 @@ let run ?(debug = false) ?ocaml_exception_info ~trace_mode file =
       write_duration_begin ~args ~thread ~name ~time;
       printf "->          END   %s\n" name
     ;;
+
+    let write_counter ~args:_ ~thread:_ ~name:_ ~time:_ : unit = printf "writing counter"
```

Comment:
We can just have this be a no-op, no print needed.

Reply:
Now a no-op.